### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by constantinoschristoforou**

### CODE REVIEW

File: .idea/cake_config_setting_v0_5.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mamchenkov) [inline .idea/cake_config_setting_v0_5.xml]: @constantinoschristoforou please configure your IDE to NOT create files like this.  Remove this please.

File: classes/class.post-type.wp-call-to-action.php [hunk 2]
```diff
@@ -1 +1 @@
...

			add_post_meta( $cta_id, 'wp-cta-ab-variation-impressions-'.$variation_id , 0 , true ) or update_post_meta( $cta_id, 'wp-cta-ab-variation-impressions-'.$variation_id , 0 );
			add_post_meta( $cta_id, 'wp-cta-ab-variation-conversions-'.$variation_id , 0 , true ) or update_post_meta( $cta_id, 'wp-cta-ab-variation-conversions-'.$variation_id , 0 );

		}
	}

	/* Load Post Type Pre Init */
	$GLOBALS['CTA_Call_To_Action_Post_Type'] = new CTA_Call_To_Action_Post_Type();
}
```
Comment (mamchenkov) [inline classes/class.post-type.wp-call-to-action.php]: @constantinoschristoforou what is the change here?

File: shared/shortcodes/ajax-functions.php [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+
+echo do_shortcode( $shortcode ) . $extra_content; ?>
+
+<?php // echo "<br>". $shortcode; ?>
+
+
+
+
+<?php    die();
```
Comment (mamchenkov) [inline shared/shortcodes/ajax-functions.php]: @constantinoschristoforou is this necessary?

File: shared/shortcodes/inbound-shortcodes.php [hunk 4]
```diff
@@ -750,14 +750,19 @@ static function inbound_forms_header_area()
 						</div>
 
 						<div id="inbound-shortcodes-preview-wrap">
-							<div id="inbound-shortcodes-preview-head">
+							<div  id="inbound-shortcodes-preview-head">
```
Comment (mamchenkov) [inline shared/shortcodes/inbound-shortcodes.php]: @constantinoschristoforou and this is needed why?

File: shared/shortcodes/js/shortcodes.js [hunk 5]
```diff
@@ -295,13 +300,14 @@
 			                   });
 						jQuery('body').trigger("inbound_forms_data_ready"); // Trigger custom hook
 
+
```
Comment (mamchenkov) [inline shared/shortcodes/js/shortcodes.js]: @constantinoschristoforou what's with the empty lines?

File: shared/shortcodes/js/shortcodes.js [hunk 6]
```diff
@@ -295,13 +300,14 @@
...
 			            }
 			    },
 				update_fields : function () {
-								var insert_form = jQuery("#inbound_shortcode_insert_default").val();
+                    var insert_form = jQuery("#inbound_shortcode_insert_default").val();
 								var current_code = jQuery("#inbound_current_shortcode").val();
 								if ( current_code === "quick_insert_inbound_form_shortcode") {
-									return false;
+                                    return false;
```
Comment (mamchenkov) [inline shared/shortcodes/js/shortcodes.js]: @constantinoschristoforou please format nicely

File: shared/shortcodes/ajax-functions.php (hunk 3 above)
Comment (constantinoschristoforou) [inline shared/shortcodes/ajax-functions.php]: The die function?

**Pull request merged**